src/client/components/app/VaultDetail/VaultDetailPanels.tsx

```diff
@@ -197,30 +208,56 @@ const VaultOverview = styled(Card)`
     }
   }
 `;
+
 export interface VaultDetailPanelsProps {
   selectedVault: GeneralVaultView;
   chartData?: any;
   chartValue?: string;
   displayAddToken?: boolean;
+  currentNetwork?: Network;
+  blockExplorerUrl?: string;
 }
 
 export const VaultDetailPanels = ({
   selectedVault,
   chartData,
   chartValue,
   displayAddToken,
+  currentNetwork,
+  blockExplorerUrl,
 }: VaultDetailPanelsProps) => {
   const { t } = useAppTranslation('vaultdetails');
 
   const isVaultMigratable = selectedVault.migrationAvailable;
   const [selectedTab, setSelectedTab] = useState(isVaultMigratable ? 'migrate' : 'deposit');
   const strategy: StrategyMetadata | null = selectedVault?.strategies[0] ?? null;
   const context = useContext(AppContext);
-
   const handleTabChange = (selectedTab: string) => {
     setSelectedTab(selectedTab);
   };
 
+  const handleScanSiteExplorer = () => {
```

Comment:
would be nice to have a ScanNetworkIcon component with all this logic, and have a prop called "network" where you sent the currentNetowork @TacosTonight , so we only import the icons and this logic there